pkg/reconciler/revision/resources/deploy.go

```diff
@@ -170,6 +176,26 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, tracingC
 	return podSpec, nil
 }
 
+func verifySecrets(rev *v1alpha1.Revision, secretLister v1.SecretLister) *apis.FieldError {
+	var apiErrs *apis.FieldError
+	for i, volume := range rev.Spec.Volumes {
+		if secret := volume.VolumeSource.Secret; secret != nil {
+			if _, err := secretLister.Secrets(rev.Namespace).Get(secret.SecretName); err != nil {
+				newErr := (&apis.FieldError{
+					Message: err.Error(),
+					Paths:   []string{"secretName"},
+				}).ViaFieldIndex("volumes", i).ViaField("spec.template.spec")
```

Comment:
Hm, this doesn't seem entirely true. This path would match for configurations and services I suppose but in this case it'd just be `spec`?